docs/operator_guides/0_running_an_operator.md

```diff
@@ -9,8 +9,8 @@ The following is the list of the verifiers currently supported by Aligned:
 
 - :white_check_mark: gnark - Groth16 (with BN254)
 - :white_check_mark: gnark - Plonk (with BN254 and BLS12-381)
-- :white_check_mark: SP1
-- :white_check_mark: Risc0
+- :white_check_mark: SP1 (v1.0.8-testnet)
+- :white_check_mark: Risc0 (v1.0.1)
```

Comment:
Maybe we should add a link to the release

Reply:
Fixed in [3f314d7](https://github.com/yetanotherco/aligned_layer/pull/566/commits/3f314d724c1dd1e7e461a1d2f1d47ef5520c27fc)